docs/add-ons/nginx.md

```diff
@@ -26,10 +26,9 @@ Note that the ingress controller is deployed in the `kube-system` namespace.
 
```

Comment:
It's useful to demonstrate NGINX Ingress with example.  We can raise another PR to cover this.
Create `ingress-controllers` folder `examples`,  add folders for  `nginx` `traefik` and `aws lb` and add exmaples

Reply:
Sure!